web/client/utils/ConfigUtils.js

```diff
@@ -255,6 +257,12 @@ var ConfigUtils = {
 
         retina: retina
         };
+    },
+    setBingKey: function(layer) {
```

Comment:
We should use more generic function that assign defaults values to the layers, or something like that